docs/tutorial/anonlink-novt.ipynb

```diff
@@ -0,0 +1,942 @@
+   "metadata": {},
+   "source": [
+    "## Get Ground Truth"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "we have 1649 true matches\n",
+      "((0, 159), (1, 169), (2, 169))\n",
+      "((1, 2309), (2, 2137))\n",
+      "((0, 366), (1, 589), (2, 362))\n",
+      "((0, 3719), (1, 3701), (2, 1560))\n",
+      "((0, 182), (1, 758), (2, 760))\n",
+      "((0, 3886), (1, 3865), (2, 311))\n",
+      "((0, 2878), (2, 280))\n",
+      "((0, 498), (1, 2269), (2, 492))\n",
+      "((0, 3630), (2, 282))\n",
+      "((1, 692), (2, 374))\n"
+     ]
+    }
+   ],
+   "source": [
+    "truth = []\n",
+    "\n",
+    "for party in [0, 1, 2]:\n",
+    "    df = pd.read_csv('data/ncvr_numrec_5000_modrec_2_ocp_20_myp_{}_nump_10.csv'.format(party))\n",
+    "    truth.append(pd.DataFrame({'id{}'.format(party): df.index, 'recid': df['recid']}))\n",
+    "    \n",
+    "dfj = truth[0].merge(truth[1], on='recid', how='outer')\n",
+    "for df in truth[2:]:\n",
+    "    dfj = dfj.merge(df, on='recid', how='outer')\n",
+    "\n",
+    "dfj = dfj.drop(columns=['recid'])\n",
+    "true_matches = set()\n",
+    "for row in dfj.itertuples(index=False):\n",
+    "    cand = [(i, int(x)) for i, x in enumerate(row) if not math.isnan(x)]\n",
+    "    if len(cand) > 1:\n",
+    "        true_matches.add(tuple(cand))\n",
+    "\n",
+    "print(f'we have {len(true_matches)} true matches')\n",
+    "e = iter(true_matches)\n",
+    "for i in range(10):\n",
+    "    print(next(e))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve with Anonlink"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Combining CLKs and Blocks for Party 0\n",
+      "Combining CLKs and Blocks for Party 1\n",
+      "Combining CLKs and Blocks for Party 2\n"
+     ]
+    }
+   ],
+   "source": [
+    "clk_files = ['novt_clk_{}.json'.format(x) for x in range(3)]\n",
+    "block_files = ['novt_blocks_{}.json'.format(x) for x in range(3)]\n",
+    "\n",
+    "clk_blocks = []\n",
+    "\n",
+    "for i, (clk_f, block_f) in enumerate(zip(clk_files, block_files)):\n",
+    "    print('Combining CLKs and Blocks for Party {}'.format(i))\n",
+    "    clk_blocks.append(combine_clks_blocks(open(clk_f, 'rb'), open(block_f, 'rb')))\n",
+    "    \n",
+    "    \n",
+    "clk_groups = []\n",
+    "rec_to_blocks = {}\n",
+    "\n",
+    "for i, clk_blk in enumerate(clk_blocks):\n",
+    "    clk_groups.append(deserialize_filters([r[0] for r in clk_blk]))\n",
+    "    rec_to_blocks[i] = {rind: clk_blk[rind][1:] for rind in range(len(clk_blk))}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assess Precision and Recall"
```

Comment:
You should explain what you mean by precision and recall, because in the multi-party case it is not obvious how you compute the true positives.